bin/spryker-sdk.sh

```diff
@@ -30,6 +30,13 @@ export DOCKER_BUILDKIT=1
 export COMPOSE_DOCKER_CLI_BUILD=1
 export EXECUTABLE_FILE_PATH=$(realpath "$0")
```

Comment:
Isn't there a way to go without `realpath`? Does this `$(cd "$0";pwd)` work for you?

Reply:
It works, thanks for your comment

Reply:
No, it doesn't work. 
I got the error `bin/spryker-sdk.sh: line 31: cd: bin/spryker-sdk.sh: Not a directory`

Reply:
Fixed